app/views/layouts/application.html.erb

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    # These elements must come first to consistently ensure proper document rendering.
```

Comment:
Is this a ruby comment in an HTML section?

Reply:
It is...but it's in the head element so technically fine?  I'll update it anyways.

Reply:
I wasn't sure of that. That's okay then